x-pack/plugins/actions/server/saved_objects/index.ts

```diff
@@ -32,6 +39,13 @@ export function setupSavedObjects(
       getTitle(obj) {
         return `Connector: [${obj.attributes.name}]`;
       },
+      onExport<RawAction>(
+        context: SavedObjectsExportTransformContext,
+        objects: Array<SavedObject<RawAction>>
```

Comment:
(Referring to the `getTitle` function on top, as comments are not allowed on non-diff lines...)

 Ideally, we shouldn't need the object's type to be displayed in its associated title, as it introduces redundancy in some parts of the SOM UI. 

Now, the main SOM table is only using an (EUI) icon to display the type associated with each object. Is there an icon associated with `rules` and `connectors` in the EUI library? If so, you could add them via the `management.icon` property of the types.

Reply:
@pgayvallet  Discussed with @mdefazio and we currently don't have dedicated icons for `rules` or `connectors`. If we're using the default icon, should we leave `Rules: ` and `Connectors: ` in the title?

Reply:
I guess so. This is low impact and can be changed later anyway, so I think it's fine.

Reply:
Thanks for being flexible.  Our concern is associating icons with Rules and Connectors that may also be used elsewhere for different concepts.  The viz type icons were created specifically for those types and aren't used for other things (or shouldn't be). 

If we want an icon, we will just need to go the route of creating on specifically for this.  However, this type column may quickly turn into a lot of different icons and likely need the title to be shown alongside them anyway.

So let's circle back to it in the future and determine if we want to _only_ show the title.